### DISCUSSION TITLE
Bootstrap

### DISCUSSION THREAD

**Discussion by kenbonsir** [termux/termux-packages#12082] [General]
Someone please share their bootstrap 64bit from TITANIUM BACKUP APP.  please anyone i will appreciate it alot. Thanks in advance.

**Comment by sylirre** [comment]
Original bootstaps are there: https://github.com/termux/termux-packages/releases/tag/bootstrap-2022.09.18-r1%2Bapt-android-7

Here is a copy of mine: [usr-aarch64.tar.gz](https://github.com/termux/termux-packages/files/9635750/usr-aarch64.tar.gz)

You don't need a full titanium backup for manually bootstrapping $PREFIX.
